recipes/server_conf.rb

```diff
@@ -22,7 +22,7 @@
   owner "postgres"
   group "postgres"
   mode 0600
-  notifies change_notify, 'service[postgresql]', :delayed
+  notifies change_notify, 'service[postgresql]', :immediately
```

Comment:
This is a direct change to the current expected behavior. Can you explain a little more why you chose to switch to `:immediately`? (and also at line 33)